projects/slap_manipulation/src/slap_manipulation/env/stretch_manipulation_env.py

```diff
@@ -0,0 +1,80 @@
+from typing import Dict
+
+import numpy as np
+import trimesh
+
+from home_robot.motion.stretch import (
+    STRETCH_BASE_FRAME,
+    STRETCH_GRASP_FRAME,
+    HelloStretch,
+    HelloStretchIdx,
+)
+from home_robot.utils.point_cloud import show_point_cloud
+from home_robot_hw.env.stretch_abstract_env import StretchEnv
+
+
+class StretchManipulationEnv(StretchEnv):
```

Comment:
we should really be using the stretch pick and place env for this right?

Reply:
I am going to merge this version as is. We can make that change in next iteration of the PR.

Reply:
file an issue for it then

Reply:
actually - i dont think you should do this though, i think you should move it into home_robot_hw as a seperate (continuous) env, as mentioned below. and that can be in this PR since its just a move